### ISSUE TITLE
Moves in the wrong direction when line is vertical

### ISSUE THREAD

**Issue by transoceanic2000** [otm/svg.math.js#1]
If you use line.move() and the source and destination points are in a vertical line (i.e. both have the same x co-ordinate) then it moves in the wrong direction.

Example:
alert( new SVG.math.Line( new SVG.math.Point( 100, 100 ),
                                       new SVG.math.Point( 100, 200 ) ).move( 
                                            new SVG.math.Point( 100, 100 ),
                                            new SVG.math.Point( 100, 200 ), 10 ) .y );

Returns 90 (i.e. moving up) but should return 110 (moving down, along the line).

**Comment by transoceanic2000** [comment]
Simplest fix is just to change the line

``` javascript
var sign = (from.x > towards.x) ? -1 : 1;
```

to

``` javascript
var sign = (from.x >= towards.x) ? -1 : 1;
```

(As the function requires a line object to work on might be nicer if it just took the distance and moved along itself that distance).

**Comment by otm** [comment]
The provided fix does not work in all cases. Sorry for missing that edge case. I will write a patch as soon as possible. Without testing I think it should look like this instead.

```
var sign = (from.x > towards.x) ? -1 :
    from.x < towards.x ? 1 : 
    from.y < towards.y ? -1 : 1;
```

**Comment by otm** [comment]
Fixed in version 0.1.1

**Comment by otm** [comment]
There is still a bug in this function.

```
    var sign = (from.x > towards.x) ? -1 :
            from.x < towards.x ? 1 : 
            from.y > towards.y ? 1 : 1; // The special case when from.x == towards.x
```

Need to add more test cases and fix...

**Comment by otm** [comment]
Fixed in 7cd41757f66104ac40cc1199e2882989c93a49f0
